frontendUiTable/src/main/java/com/virtuslab/gitmachete/frontend/ui/table/GitMacheteGraphTable.java

```diff
@@ -146,14 +156,15 @@ public GraphTableModel getModel() {
   public Object getData(String dataId) {
     var gitMacheteRepository = gitMacheteRepositoryRef.get();
     return Match(dataId).of(
+        typeSafeCase(DataKeys.KEY_BRANCH_LAYOUT, branchLayout),
```

Comment:
Do we need to do anything special to protect ourselves from `branchLayout`/`macheteFilePath` being null or is it taken care of already?

Reply:
Actually we do not need to do anything because the setting of the branch layout and machete path happens together with a repository update. The eventually git machete repository existence is equivalent to the fact that these variables have been properly provided and as `git machete repository is ready`, they can be safely used by actions. On the other hand, we do not set them to `null` in the case of failures in `GitMacheteRepositoryManager` but they won't be used anyway because `git machete repository is not ready`.